libbeat/docs/queueconfig.asciidoc

```diff
@@ -22,16 +22,20 @@ queue.mem:
 === Configure the memory qeueue
 
 The memory queue keeps all events in memory. It is the only queue type
-supported right now.  By default no flush interval is configured. All events
-published to this queue will be directly consumed by the outputs.
+supported right now. 
+
+If no flush interval and no number of events to flush is configured, 
+all events published to this queue will be directly consumed by the outputs.
+To enforce spooling in the queue, set the `flush.min_events` and `flush.timeout` options.
+
+By default `flush.min.events` is set to 1024 and `flush.timeout` is set to 1s.
```

Comment:
I like the new paragraph shortly explaining implications of these settings. Only: `s/flush.min.events/flush.min_events` , `s/1024/2048/`.